src/expr/src/scalar/mod.rs

```diff
@@ -1402,6 +1405,8 @@ impl MirScalarExpr {
         }
     }
 
+    /// swap_remove the elements indicated by `indexes`.
+    /// `indexes` shouldn't have duplicates.
```

Comment:
Nit: Improve doc comment to say that `swap_remove_multiple` will panic if `indexes` has duplicates. The semantics of the function aren't immediately clear from the description, maybe this would be a good opportunity to explain what the function does?

Reply:
done